src/base/PortalContext.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, ReactNode, useState } from 'react';
+
+export interface PortalContextData {
+  portalElement: HTMLDivElement | null;
+  setPortalElement: ((element: HTMLDivElement) => void) | null;
+}
+
+export const portalContextDefaultValues = {
+  portalElement: null,
+  setPortalElement: null
+};
+
+export const PortalContext = createContext<PortalContextData>(
```

Comment:
From what I understand, it's up to the user of RMWC to use this context if needed. Do you think you'd be able to add some docs for this?

Reply:
Yes, that is correct. I'd be happy to put some information together for the docs. Where at would you recommended this be added? A new page?

Reply:
That would be ideal I think. If you need help with that, you can write some docs and send it to me and I can place it on the site.